### ISSUE TITLE
Regexr not loading all characters correctly in v3

### ISSUE THREAD

**Issue by tomhumphrey** [gskinner/regexr#219]
Entered a string into the old RegeExr
```
Test A1 (III’15) 270     10/12  ABC/DEF       PNR       AVC
Test Asd(II’05) 300     11/12  RtF/ZXC      PNR        NKL
Test 33 (I’01) PIL     11/12  KNP/ILO      IL 90.5    FX - NO
Test 4 (IIII’10)   270  11-12/12  JKI/IOP   PNR      RPTD - RPTD
```
And saved the example. But then when going to load it instead loads
```
Test A1 (IIIâ€™15) 270     10/12  ABC/DEF       PNR       AVC
Test Asd(IIâ€™05) 300     11/12  RtF/ZXC      PNR        NKL
Test 33 (Iâ€™01) PIL     11/12  KNP/ILO      IL 90.5    FX - NO
Test 4 (IIIIâ€™10)   270  11-12/12  JKI/IOP   PNR      RPTD - RPTD
```

[Link for demo](https://regexr.com/3hcn7)

**Comment by gskinner** [comment]
Good catch. It looks like this is only an issue when patterns saved in v2 with a few specific characters are loaded into v3. Saving the same characters in v3 works fine.

We're looking to see if there's a way to fix the encoding on old patterns, without causing problems with new ones.

**Comment by wdamien** [comment]
Thanks for the report! This turned out to be an UTF-8 encoding issue, so we were able to fix all the old saves with the 3.1 update.
